### ISSUE TITLE
Conference Comfort

### ISSUE THREAD

**Issue by linaori** [symfony/diversity#78]
The idea of this issue is to aggregate small ideas that would add a little comfort with minimal effort.

- **Squeezable lemons** to add taste to water. This adds a bit of variety when drinking only water.

**Comment by lsmith77** [comment]
https://github.com/symfony/diversity/issues/46

**Comment by romaricdrigon** [comment]
Hello,
I'm not sure if it should be there of if there's some food specific issue (I couldn't find it), but having labels on food would help greatly. To start, it could be only the main ingredients. It would already help immensely people who want to avoid meat or gluten, by choice or intolerance. This is something important to me, as I have some uncommon allergies and I've met at previous conferences vegetarian persons who were clueless about which snacks they could eat.
Ultimately, having an exhaustive list of ingredients with allergens highlighted would be the best but I understand this is challenging logistically.

**Comment by linaori** [comment]
@romaricdrigon Yes, this is a very important point. There was an issue at the hackathon that highlighted the importance of ingredients being mentioned.

This particular issue regarding food and such, is already something we listed (among several) in a document that I've gathered from a twitter feed. We're still in the process of working through this document to deduplicate the raw data, but will definitely share it when done.

I've created a copy of the document (because the original is edit by link) where you can see our current (raw) list of topics. You can find food related stuff in the bottom. For the link see #87

**Comment by romaricdrigon** [comment]
Great, thank you for the follow-up!
